vulkan_helpers/helper_functions.h

```diff
@@ -284,7 +284,7 @@ VkQueryPool CreateQueryPool(VkDevice* device,
 uint32_t inline GetMemoryIndex(VkDevice* device, logging::Logger* log,
                                uint32_t required_index_bits,
                                VkMemoryPropertyFlags required_property_flags) {
-  const VkPhysicalDeviceMemoryProperties& properties =
+    const VkPhysicalDeviceMemoryProperties& properties =
```

Comment:
nit: this change doesn't looks consistent with rest of indent style - was this change an accident?

Reply:
Likely an accident I will fix.